examples/flutter_gallery/lib/demo/fortnightly/fortnightly.dart

```diff
@@ -0,0 +1,229 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+class FortnightlyDemo extends StatelessWidget {
+  static const String routeName = '/fortnightly';
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Typography Demo',
+      theme: _fortnightlyTheme,
+      home: Scaffold(
+        body: Stack(
+          children: <Widget>[
+            FruitPage(),
+            Padding(
+              padding: EdgeInsets.only(top: MediaQuery.of(context).padding.top),
+              child: ShortAppBar(onPressed: (){
+                Navigator.pop(context);
+              },),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class ShortAppBar extends StatelessWidget {
+  const ShortAppBar({this.onPressed});
+
+  final VoidCallback onPressed;
```

Comment:
I think calling this something like `onBackPressed` might be a little clearer

Reply:
Done.